src/views/quiz-settings/quiz-settings.js

```diff
@@ -0,0 +1,138 @@
+import './quiz-settings.css';
+import '../../style.css';
+import Button from '../../components/Button/Button';
+
+class QuizSettings {
```

Comment:
to be honest i do not like the way it is implemented with static methods, seems overengineered for me